src/test/java/seedu/resireg/logic/CommandHistoryTest.java

```diff
@@ -45,6 +45,15 @@ void getHistory() {
         assertEquals(Arrays.asList("c", "d"), historyWithCD.getHistory());
     }
 
+    @Test
+    void getCounter() {
+        final CommandHistory historyWithAC = new CommandHistory();
```

Comment:
Thanks for pointing this out, but I don't think not defining the objects inside `CommandTestUtils` will cause inconsistency, since such testing is done by classes under the Model package. That said, I think I can create a class to contain typical histories.